src/components/Garden/ModalFlows/SupportProposal/SupportProposal.js

```diff
@@ -1,5 +1,5 @@
 import React, { useCallback, useMemo, useState } from 'react'
-import BigNumber from '../../../lib/bigNumber'
+import BigNumber from '../../../../lib/bigNumber'
```

Comment:
Let's also move it to where the `@utils` imports are

```suggestion
import BigNumber from '@lib/bigNumber'
```